mozci/push.py

```diff
@@ -1146,11 +1151,16 @@ def __make_group_summary_objects(pushes):
 
     for task_id, result_group, result_duration in results:
         # Obtain TestTask object that matches the task_id.
-        task = [t for push in pushes for t in push.tasks if t.id == task_id].pop()
+        tasks = [t for push in pushes for t in push.tasks if t.id == task_id]
+        if len(tasks) == 0:
+            continue
+        task = tasks.pop()
 
+        parts = task.label.split('/')
+        key = "%s-%s:%s" % (parts[0], parts[1].split('-')[0], result_group)
```

Comment:
If you want to get the task configuration, you can just use `task.configuration` instead